index.js

```diff
@@ -113,7 +113,7 @@ function makeReporter(callback, arr) {}
  *  3. `reporter`
  *
  * 1. call checkQueue();
- * 2. if the queue is empty (i.e. [0, 0]) call the callback function and return.
+ * 2. if the queue is empty (i.e. [0, 0]) call the callback/reporter function and return.
```

Comment:
Let's remove callback.

```suggestion
 * 2. if the queue is empty (i.e. [0, 0]) call the reporter function and return.
```